test/jasmine/karma.conf.js

```diff
@@ -323,13 +319,6 @@ if(isBundleTest) {
             func.defaultConfig.files.push(constants.pathToPlotlyDistMin);
             func.defaultConfig.preprocessors[testFileGlob] = ['browserify'];
             break;
-        case 'ie9':
-            // load ie9_mock.js before plotly.js+test bundle
-            // to catch reference errors that could occur
-            // when plotly.js is first loaded.
-            func.defaultConfig.files.push(pathToIE9mock);
-            func.defaultConfig.preprocessors[testFileGlob] = ['browserify'];
-            break;
```

Comment:
👋 IE9 💯 
Another good reason to make the next release a 2.0 though 😎

Reply:
IE9 (and IE10) have already been broken for quite a time.
I don't think we have to hold this feature until v2.

Reply:
Here is more detailed background on IE10.
As mentioned [here](https://github.com/browserify/browserify/releases/tag/v16.4.0) `browserify-v16.4.0` drop the support for IE10.
867e43c319edb5158a516d3704c6c8d95c1ef05c was committed back in September 2019.
So the last `plotly.js` version in which IE9 & IE10 were supported should be v1.49.5; and I don't recall a single issue opened related to that.

Reply:
Outdated. Resolving...